### PR TITLE
GroupBy now properly works with valueMember and dataMember

### PR DESCRIPTION
**Pull request by sglogovic**
ValueMember in quickGrid is used for sort/group value while dataMember for display. GroupBy is now properly shows dataMember while grouping

### CODE REVIEW

File: src/components/QuickGrid/DataSelectors.ts [hunk 1]
```diff
@@ -47,6 +47,6 @@ const getSortedRows = createSelector(getInputRows, getSortColumn, getSortDirecti
         return sortRows(rows, sortColumn, sortDirection, groupBy, columns);
     });
 
-export const getRowsSelector = createSelector(getSortedRows, getGroupBy, getExpandedRows, (rows, groupedColumns, expandedRows = {}) => {
-    return groupRows(rows, groupedColumns, expandedRows);
+export const getRowsSelector = createSelector(getSortedRows, getGroupBy, getExpandedRows, getColumns, (rows, groupedColumns, expandedRows, columns = {}) => {
```
Comment (MatijaA) [inline src/components/QuickGrid/DataSelectors.ts]: is columns = {} intentionally ?

Comment (sglogovic) [reply src/components/QuickGrid/DataSelectors.ts]: No, it should stay with expandedRows

File: src/components/QuickGrid/rowGrouper.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import * as _ from 'lodash';
-import { GroupRow, IGroupBy } from './QuickGrid.Props';
+import { GroupRow, IGroupBy, GridColumn } from './QuickGrid.Props';
 class RowGrouper {
     groupByColumns: Array<IGroupBy>;
+    columns: Array<GridColumn>;
     expandedRows: any;
-        constructor(groupByColumns, expandedRows) {
+        constructor(groupByColumns, expandedRows, columns) {
         this.groupByColumns = groupByColumns.slice(0);
```
Comment (hbagarich) [inline src/components/QuickGrid/rowGrouper.ts]: We raised the TypeScript version, so we can use spread operator instead of slice, [...groupByColumns]

Comment (sglogovic) [reply src/components/QuickGrid/rowGrouper.ts]: Replaced

**Pull request merged**